### PR TITLE
fix incorrect gpu parameter handling in pipeline parser

### PR DESCRIPTION
**Pull request by wenjun93**
### How was this pull request tested?

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by lresende — APPROVED** [review]

**Comment by lresende** [comment]
@wenjun93 How about CPU, should we leave it as request_cpu? or should we use cpu_limit (see [docs](https://airflow.apache.org/docs/apache-airflow/1.10.12/_api/airflow/contrib/operators/kubernetes_pod_operator/index.html))

**Comment by wenjun93** [comment]
@lresende gpu limit is mandatory, while cpu_limit is optional. BTW I agress we should expose cpu/mem requests options to front end.

File: elyra/templates/airflow/airflow_template.jinja2 [hunk 1]
```diff
@@ -68,7 +68,7 @@ op_{{ operation.id|regex_replace }} = KubernetesPodOperator(name='{{ operation.n
                                                                        'request_memory': '{{ operation.mem_request }}',
             {% endif %}
             {% if operation.gpu_limit %}
-                                                                       'limit_gpu': '{{ operation.cpu_request }}',
+                                                                       'limit_gpu': '{{ operation.gpu_limit }}',
```
Comment (kevin-bates) [inline elyra/templates/airflow/airflow_template.jinja2:71]: Nice catch!

**Review by kevin-bates — APPROVED** [review]

**Pull request merged**